specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                },
+                "jobACLs": {
+                    "$ref": "#/definitions/JobACLs",
+                    "description": "The ACLs of the job."
+                },
+                "jobComplete": {
+                    "type": "boolean",
+                    "x-nullable": false,
+                    "description": "Whether or not the job has completed."
+                },
+                "jobFile": {
+                    "type": "string",
+                    "description": "The job configuration file."
+                },
+                "jobId": {
+                    "type": "string",
+                    "description": "The full ID of the job."
+                },
+                "jobID": {
+                    "$ref": "#/definitions/JobID",
+                    "description": "The ID of the job."
+                },
+                "jobName": {
+                    "type": "string",
+                    "description": "The user-specified job name."
+                },
+                "jobPriority": {
+                    "type": "string",
+                    "description": "The priority of the job."
+                },
+                "mapProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the maps."
+                },
+                "neededMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory needed for the job."
+                },
+                "numReservedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots reserved."
+                },
+                "numUsedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots used for the job."
+                },
+                "priority": {
+                    "type": "string",
+                    "description": "The priority of the job."
+                },
+                "queue": {
+                    "type": "string",
+                    "description": "The job queue name."
+                },
+                "reduceProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the reduces."
+                },
+                "reservedMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory reserved for the job."

```

Comment:
What is the unit?

Reply:
Not sure. Keep parity with old SDK. See code [here](https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/HDInsightJob/HDInsightJob/Generated/Models/Status.cs#L232-L241).

---
In reply to: [237732378](https://github.com/Azure/azure-rest-api-specs/pull/4489#discussion_r237732378) [](ancestors = 237732378)